### ISSUE TITLE
Offer share function

### ISSUE THREAD

**Issue by PanderMusubi** [CNugteren/NLWeer#11]
In the main menu, offer a share function to share (the url of) the weather map you are looking at via Android share API into WhatsApp, Telegram, Signal, SMS, e-mail, etc.

When possible, offer the same functionality to share weather text or a weather graph.

Only when it really looks nice and is not in the way of any content, consider adding a Material Design share button as seen below.

![Screenshot_20200418-164228_NLWeer](https://user-images.githubusercontent.com/198964/79640916-b03aae00-8194-11ea-9a88-f9636f06b3f9.png)

**Comment by CNugteren** [comment]
OK, makes sense. I'm not actively working on the app at the moment, but I will implement this when I work on it again. In the meantime, others should feel free to add such an option (I would suggest to do it just as an extra menu item though) and submit a PR.

**Comment by CNugteren** [comment]
I've now added a share functionality. Currently it does only the URL of the map in view (so not the text or the chart). And I've placed it in the menu on the top right to not clutter the screen too much.

Here's what it looks like:

![Screenshot_20200717-223018](https://user-images.githubusercontent.com/1162108/87829031-71dd8980-c87e-11ea-83cb-604662ead264.png)
![Screenshot_20200717-223034](https://user-images.githubusercontent.com/1162108/87829033-72762000-c87e-11ea-89ec-a89ae52bd4db.png)

This is now in master and will be part of the next NLWeer release.
